### PR TITLE
[ko] Korean Translation for 3-transport lesson 2

### PR DESCRIPTION
**Pull request by jhYun505**
Add README.ko.md

### CODE REVIEW

File: 3-transport/lessons/2-store-location-data/translations/README.ko.md [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+      ```sh
+      az storage account keys list --output table \
+                                   --account-name <storage_name>
+      ```
+
+      `<storage_name>`을 스토리지 계정의 이름으로 수정합니다.
+
+      `key1`의 값을 복사합니다.
+
+   1. 다음 명령어를 실행하여 컨테이너읭 Blob을 나열합니다.
```
Comment (WZNT-KimHyeJin) [inline 3-transport/lessons/2-store-location-data/translations/README.ko.md:385]: 읭  ㅇㅡ ㅇ

File: 3-transport/lessons/2-store-location-data/translations/README.ko.md [hunk 2]
```diff
@@ -0,0 +1,471 @@
...
+
+      ```sh
+      az storage blob download --container-name gps-data \
+                               --account-name <storage_name> \
+                               --account-key <key1> \
+                               --name <blob_name> \
+                               --file <file_name>
+      ```
+
+      Replace `<storage_name>` with the name of the storage account, and `<key1>` with the value of `key1` you copied in the earlier step.
```
Comment (WZNT-KimHyeJin) [inline 3-transport/lessons/2-store-location-data/translations/README.ko.md:417]: You have to translate this line!

File: 3-transport/lessons/2-store-location-data/translations/README.ko.md [hunk 3]
```diff
@@ -0,0 +1,471 @@
...
+      `<storage_name>`를 스토리지 계정의 이름으로 바꿉니다. `<key1>` 값을 이전 단계에서 복사한 `<key1>` 값으로 바꿉니다.
+
+      Replace `<blob_name>` with the full name from the `Name` column of the output of the last step, including the folder name. Replace `<file_name>` with the name of a local file to save the blob to.
```
Comment (WZNT-KimHyeJin) [inline 3-transport/lessons/2-store-location-data/translations/README.ko.md:420]: Plz remove this line!

**Review by WZNT-KimHyeJin — COMMENTED** [review]
I love your translation! You did great !! 🍀🍀

File: 3-transport/lessons/2-store-location-data/translations/README.ko.md (hunk 2 above)
Comment (jhYun505) [reply 3-transport/lessons/2-store-location-data/translations/README.ko.md]: Thanks

**Comment by WZNT-KimHyeJin** [comment]
@jimbobbennett She finished the translation! Plz merge this!

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**